extras/AFC_prep.py

```diff
@@ -94,7 +95,13 @@ def PREP(self, gcmd):
                 if 'color' not in self.AFC.lanes[LANE.unit][LANE.name]: self.AFC.lanes[LANE.unit][LANE.name]['color']='#000000'
                 if 'weight' not in self.AFC.lanes[LANE.unit][LANE.name]: self.AFC.lanes[LANE.unit][LANE.name]['weight'] = 0
                 if 'runout_lane' not in self.AFC.lanes[LANE.unit][LANE.name]: self.AFC.lanes[LANE.unit][LANE.name]['runout_lane']='NONE'
-                if 'map' not in self.AFC.lanes[LANE.unit][LANE.name] or self.AFC.lanes[LANE.unit][LANE.name]['map'] is None: self.AFC.lanes[LANE.unit][LANE.name]['map'] = LANE.map
+                if 'map' not in self.AFC.lanes[LANE.unit][LANE.name]:
+                   self.AFC.lanes[LANE.unit][LANE.name]['map'] = 'NONE'
+                else:
+                   LANE.map = self.AFC.lanes[LANE.unit][LANE.name]['map']
+                if LANE.map != 'NONE':
```

Comment:
This if statement is not needed same thing with line 103 since its just reassigning the value back to the array that was assigned to `LANE.map` in the else statement above, just need to keep line 104 within the else statement